frappe/email/doctype/email_account/email_account.js

```diff
@@ -167,21 +169,25 @@ frappe.ui.form.on("Email Account", {
 			delete frappe.route_flags.delete_user_from_locals;
 			delete locals["User"][frappe.route_flags.linked_user];
 		}
-	},
 
-	after_save(frm) {
-		if (frm.doc.auth_method === "OAuth" && !frm.doc.refresh_token) {
-			oauth_access(frm);
+		if (frm.doc.connected_app && !frm.doc.connected_user) {
```

Comment:
any better way to set session user in user link field?

Reply:
Related question: would an account need to be authorized by administrator for background pulls to succeed?

Reply:
I am assuming same functionality as current Email Account where System Manager adds an `Email Account` or `Email Domain` with needed addresses and ports and sets awaiting password so the user can just fill up their password and continue.

In OAuth, System Manager maps `Connected App` and Frappe `User` in `Email Account`. User clicks authorize on Email Account and logins in with account on Connected App (Google/Microsoft/Examplemail).

Do we need Admin to further do anything?